src/Filters/DateRangeFilter.php

```diff
@@ -157,13 +155,15 @@ public function useColumn(string $column): self
         return $this;
     }
 
-    public function dateRangeQuery(Builder $query, array $data = []): Builder
```

Comment:
Why was this changed?

Reply:
To overload `apply` function from trait `InteractsWithTableQuery`, this follows how Filaments plugins work.

If the developer needs to further modify the query, it would need to mimic some of the parent `apply` code (mainly checking `$this->hasQueryModificationCallback()`.